.github/workflows/ci.yml

```diff
@@ -1,53 +1,29 @@
 name: CI
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - main
+      - master
+  pull_request:
 
 jobs:
-  tests:
+  test:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        ruby: [2.4, 2.7, '3.0', 3.1, 3.2, truffleruby-head]
-
+        ruby: ["2.7", "3.0", "3.1", "3.2", "3.3"]
+    name: Ruby ${{ matrix.ruby }}
     steps:
-    - uses: actions/checkout@v3
-
-    - name: Sets up the Ruby version
-      uses: ruby/setup-ruby@v1
-      with:
-        ruby-version: ${{ matrix.ruby }}
-
-    - name: Sets up the environment
-      run: |
-        sudo apt-get install libsqlite3-dev
-
-    - name: Install legacy bundler for Ruby 2.4
-      if: ${{ matrix.ruby == 2.4 }}
-      run: |
-        gem install -q bundler -v 2.3.26 
-
-    - name: Install bundler 2.4+ for modern Rubies
-      if: ${{ matrix.ruby != 2.4 }}
-      run: |
-        gem install -q bundler
-
-    - name: Run bundle install
-      run: |
-        bundle install
+      - uses: actions/checkout@v2
 
-    - name: Runs code QA and tests
-      run: bundle exec rake
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+          bundler-cache: true
 
-    - name: Publish to Rubygems
```

Comment:
So publishing this gem was a part of CI? or merging to master? and now we are moving it to be done manually locally right?

Reply:
the upstream gem had this setup, but I didn't wire this up for our gem forks. we could consider that follow upfor all our gems in github down the road.